Sources/Concordium/Domain/Block.swift

```diff
@@ -1,15 +1,13 @@
 import Foundation
 
-public typealias BlockHash = Data // 32 bytes
-
-public enum BlockIdentifier {
+public enum BlockIdentifier: ToGRPC {
     case lastFinal
     case best
     case hash(BlockHash)
     case absoluteHeight(UInt64)
     case relativeHeight(genesisIndex: UInt32, height: UInt64, restrictedToGenesisIndex: Bool)
 
-    func toGRPCType() -> Concordium_V2_BlockHashInput {
+    func toGRPC() -> Concordium_V2_BlockHashInput {
```

Comment:
same here `Concordium_V2_BlockHashInput` its not swift naming convention. we use camelcase instead. 
`toGRPC` converts to `Concordium_V2_BlockHashInput` its not obvious why here gprc, maybe better to rename this function, to make to more self explanatory

Reply:
These types (`Concordium_V2_*`) are from the code generated from the GRPC API. Also they're internal only as the generated code is not public, so no user will ever use this type when using the SDK.

Can we just keep them as is? 😄